src/Paket.PowerShell/System.Management.Automation.ps1

```diff
@@ -0,0 +1,2 @@
+# copies System.Management.Automation.dll from the GAC to this folder
+cp ([PSObject].Assembly.Location) .
```

Comment:
Powershell.exe should be invoked with exra args "-executionpolicy bypass" so that it works even for those nonbelievers like @forki who don't use powershell and still have default Restricted policy ;-)